ykbuild/build.rs

```diff
@@ -181,10 +186,6 @@ fn main() {
                 .unwrap(),
         );
 
-        if generator == "Unix Makefiles" {
```

Comment:
So does `make` get a sensible default `-j` from elsewhere now?

Reply:
Yes, we pass `-j` to *whatever* the build command is, without special casing makefiles.